app/app.go

```diff
@@ -444,16 +444,18 @@ func NewApp(
 		keys[ibctransfertypes.StoreKey],
 		ibctransferSubspace,
 		app.ibcKeeper.ChannelKeeper,
```

Comment:
Do you understand why this is passed twice (albeit once as an "Ics4Wrapper")- seems weird?

Reply:
yeah great point, this is from the migration docs for v2->v3.
> The transferkeeper.NewKeeper(...) now takes in an ICS4Wrapper. The ICS4Wrapper should be the IBC Channel Keeper unless ICS 20 is being connected to a middleware application.
So unless we are using a middleware here, we should just pass the channel keeper here.